AsyncDisplayKit/Private/ASDataController+Subclasses.h

```diff
@@ -67,9 +62,9 @@ typedef void (^ASDataControllerCompletionBlock)(NSArray<ASCellNode *> *nodes, NS
 - (void)insertSections:(NSMutableArray *)sections ofKind:(NSString *)kind atIndexSet:(NSIndexSet *)indexSet completion:(void (^)(NSArray *sections, NSIndexSet *indexSet))completionBlock;
 
 /**
- * Deletes the given sections of the specified kind in the backing store, calling completion on the main thread when finished.
+ * Deletes the given sections in the backing store, calling completion on the main thread when finished.
  */
-- (void)deleteSectionsOfKind:(NSString *)kind atIndexSet:(NSIndexSet *)indexSet completion:(void (^)(NSIndexSet *indexSet))completionBlock;
+- (void)deleteSections:(NSIndexSet *)indexSet completion:(void (^)())completionBlock;
```

Comment:
Since `ASCollectionDataController` no longer use this method and `deleteNodesOfKind:atIndexPath:completion:`, we can just go ahead and remove them this header. Not a big deal though as I will do it in my refactoring anyway.